docs/installation.md

```diff
@@ -21,31 +21,10 @@ This document ends with these appendices:
 This installation guide assumes that the following softwares are already installed on the target system:
 
 * one of CentOS, Debian, FreeBSD or Ubuntu
-* one of MySQL or PostgreSQL
 * sudo
 * curl (only for post-installation sanity check)
 
 

```

Comment:
The databases have to be introduced before the dependency section. E.g.:

"This installation guide assumes that one of the possible database backends has been selected:
* Mysql
* Postgresql
* Sqlite"